man/jail.conf.5

```diff
@@ -89,13 +89,33 @@ indicates that the specified file is to be parsed before the current file.
 indicates that the specified file is to be parsed after the current file.
 .RE
 
-Using Python "string interpolation" mechanisms, other definitions are allowed and can later be used within other definitions as %(name)s. For example.
+Using Python "string interpolation" mechanisms, other definitions are allowed and can later be used within other definitions as %(name)s. 
+Additionaly fail2ban has an extended interpolation feature named \fB%(known/parameter)s\fR (means last known option with name \fBparameter\fR). This interpolation makes possible to extend a stock filter or jail regexp in .local file (opposite to simply set failregex/ignoreregex that overwrites it). For example.
 
 .RS
+.nf
 baduseragents = IE|wget
+failregex = %(known/failregex)s
+            useragent=%(baduseragents)s
```

Comment:
But `%(known/failregex)s` should be used only in this way if possible, because `%(known/failregex)s` may contain a list of expressions (multi-line).
So if we have today in filter:
```
failregex = .* agent=test
```
And someone writes in jail
```
failregex = %(known/failregex)s|.* useragent=%(baduseragents)s
```
it will work, but ...
but tomorrow someone extend failregex with another expression (as list)
```
failregex = .* agent=test
            .* something_else=test
```
the expression in `jail` will be misleading.

Therefore I don't like an example, where the parameter `%(known/failregex)s` will be knowingly used wrong.